src/kup/__main__.py

```diff
@@ -336,7 +342,7 @@ def list_package(package_name: str, show_inputs: bool) -> None:
     else:
         table_data = [
             ['Package', 'Installed version', 'Status'],
-        ] + [[name, p.version, p.status] for name, p in packages.items()]
+        ] + [[name, f'{p.version}{" (" + p.tag + ")" if p.tag else ""}', p.status] for name, p in packages.items()]
```

Comment:
Consider extracting this expression:
```python3
tag_str = f' ({p.tag})' if p.tag else ''
```

then

```python3
f'{p.version}{tag_str}'
```

Reply:
its inside a list comprehension, can i do a local binding there?